docs/public/specifications/tests/Address-TP/address01.md

```diff
@@ -1,47 +1,154 @@
-## ADDRESS01: Name server address must be globally routable
+## ADDRESS01: Name server address must be globally reachable
 
-### Test case identifier
-**ADDRESS01** Name server address must be globally routable
+## Test case identifier
+**ADDRESS01** 
 
-### Objective
+## Table of contents
+
+* [Objective](#Objective)
+* [Scope](#Scope)
+* [Inputs](#Inputs)
+* [Summary](#Summary)
+* [Test procedure](#Test-procedure)
+* [Outcome(s)](#Outcomes)
+* [Intercase dependencies](#Intercase-dependencies)
+
+
+## Objective
 
 In order for the domain and its resources to be accessible, authoritative 
 name servers must have addresses in the routable public addressing space.
 
 IANA is responsible for global coordination of the IP addressing system.
 Aside its address allocation activities, it maintains reserved address ranges
-for special uses. These ranges can be categorized into three types : 
-[Special purpose IPv4
-addresses](https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xml),
-[Special purpose IPv6
-addresses](https://www.iana.org/assignments/iana-ipv6-special-registry/iana-ipv6-special-registry.xml)
-and [Multicast reserved
-addresses](https://www.iana.org/assignments/multicast-addresses/multicast-addresses.xml).
+for special uses. These ranges can be categorized into two types: 
+[Special purpose IPv4 addresses] and [Special purpose IPv6 addresses].
+
+This test checks the name server IP addresses, both those derived from delegation and those 
+derived from the name servers listed in the zone (NS records). Each address is compared against
+the IANA databases. If an address is not globally routable, a message is outputted.
+
+## Scope
+
+This test case does not do any  actual connectivity test, which is done by other test cases.
+
+## Inputs
+
+* "Child Zone" -- the domain name to be tested.
+
+## Summary
 
-### Inputs
+Message Tag                       | Level    | Arguments | Message ID for message tag
+:-------------------------------- |:---------|:----------|:--------------------------
+A01_ADDR_GLOBALLY_REACHABLE       | INFO     |           | All IP addresses of all name servers are listed as in the globally reachable address space.
+A01_NO_NAME_SERVERS_FOUND         | CRITICAL |           | No name servers found.
+A01_NO_GLOBALLY_REACHABLE_ADDR    | CRITICAL |           | None of the name servers IP addresses are listed as globally reachable.
```

Comment:
Maybe this should be lowered from CRITICAL to ERROR. In a local context testing might be possible. In the definition is says "A result at level CRITICAL means that the zone being tested has one or more problems that are so severe that it is not possible to even test it".

Reply:
Yes, I suggest it to be ERROR instead of CRITICAL.